### PR TITLE
New longpress and getLovelace

### PR DESCRIPTION
**Pull request by RomRider**
LongPress with dblTap + hold Repeat. (longpress still not working for me)
Added a function to retrieve the LL object.

### CODE REVIEW

File: src/handle-click.ts [hunk 1]
```diff
@@ -40,8 +40,8 @@ export const handleClick = (
           entityId: actionConfig.entity
             ? actionConfig.entity
             : config.entity
-            ? config.entity
-            : config.camera_image
+              ? config.entity
```
Comment (iantrich) [inline src/handle-click.ts]: accidental?

File: src/long-press-directive.ts [hunk 2]
```diff
@@ -1,63 +1,76 @@
-import { directive, PropertyPart } from "lit-html";
+import { directive, PropertyPart } from 'lit-html';
```
Comment (iantrich) [inline src/long-press-directive.ts]: I need to add lint configs to this repo :)

File: src/long-press-directive.ts [hunk 3]
```diff
@@ -1,63 +1,76 @@
...
-const isTouch =
-  "ontouchstart" in window ||
-  navigator.maxTouchPoints > 0 ||
-  navigator.msMaxTouchPoints > 0;
+const isTouch = 'ontouchstart' in window
+  || navigator.maxTouchPoints > 0
+  || navigator.msMaxTouchPoints > 0;
 
-interface LongPressInterface extends HTMLElement {
+interface LongPress extends HTMLElement {
```
Comment (iantrich) [inline src/long-press-directive.ts]: I don't personally like that the interface and class share the same name, hence why I purposely changed this

File: src/handle-click.ts (hunk 1 above)
Comment (RomRider) [reply src/handle-click.ts]: My formatter does it this way, it's just indentation, or am I missing smth?

File: src/long-press-directive.ts (hunk 3 above)
Comment (RomRider) [reply src/long-press-directive.ts]: Okay 👍

File: src/handle-click.ts (hunk 1 above)
Comment (iantrich) [reply src/handle-click.ts]: Well I don't want it going back and forth depending on which formatter someone is using. But that just means I need to add some linter configs 😄

**Pull request merged**